### PR TITLE
change to use isFast

### PR DESCRIPTION
**Pull request by alexzk1**
That compiled with llvm 6.0+

### CODE REVIEW

**Comment by alexzk1** [comment]
Yes, that the problem kinda I think. I need 2 different packages, but your repo for stdlib do not build pch and  compiler do. So my script just installs what compiler did there. It doesnt seem correct, but will see. Compiler shows me path to pch which was used to build compiler instead current.

>Why do you re-apply the same patch which will already be merged into the repo?
Because I clone your repo. Wanted to install stuff asap and have it fresh later.

**Comment by alexzk1** [comment]
If/when you fix your repo, I just will delete patch (or end user if any can delete). Just now I need to see if your opencl + standard opencv will boost my program. That is fastest way to move pi/desktop - pacman installer script.

File: src/precompilation/FrontendCompiler.cpp [hunk 1]
```diff
@@ -63,13 +63,15 @@ static std::string buildClangCommand(const std::string& compiler, const std::str
     // link in our standard-functions
     command.append(" -Wno-undefined-inline -Wno-unused-parameter -Wno-unused-local-typedef -Wno-gcc-compat ");
     if(usePCH)
-        command.append("-include-pch " VC4CL_STDLIB_HEADER " ");
+        //command.append("-include-pch " VC4CL_STDLIB_HEADER " ");
+        command.append("-include-pch vc4cl-stdlib/VC4CLStdLib.h.pch ");
```
Comment (doe300) [inline src/precompilation/FrontendCompiler.cpp]: This change should not be in this PR

File: src/precompilation/FrontendCompiler.cpp [hunk 2]
```diff
@@ -63,13 +63,15 @@ static std::string buildClangCommand(const std::string& compiler, const std::str
...
     else
     {
         command.append("-finclude-default-header ");
         // The #defines (esp. for extensions) from the default headers differ from the supported #defines,
         // so we need to include our #defines/undefines
-        command.append("-include " VC4CL_STDLIB_CONFIG_HEADER " ");
+        //command.append("-include " VC4CL_STDLIB_CONFIG_HEADER " ");
+        command.append("-include vc4cl-stdlib/defines.h ");
```
Comment (doe300) [inline src/precompilation/FrontendCompiler.cpp]: Same here

**Comment by alexzk1** [comment]
yes, sorry. Well, u know where and what to change to compile with llvm6+ 
I need something working to finish my calcs on gpu.

**Pull request closed without merging**